src/main/java/com/cosc436002fitzarr/brm/enums/Status.java

```diff
@@ -0,0 +1,7 @@
+package com.cosc436002fitzarr.brm.enums;
+
+public enum Status {
```

Comment:
Please add an `Overdue` value here so we can keep track of when the associate has been assigned to the building risk assessment but hasn't completed it within the specified due date.